src/components/ObjectSearch.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="objectSearch">
-    <VueSelect label="name" :options="objects" v-model="tmpSelect" :on-change="selectObject">
+    <VueSelect label="name" :options="sortedObjects" v-model="selectedObject" :on-change="selectObject">
```

Comment:
Just trying to remember why I didn't bind the model straight to the prop. I think I was getting an error but can't remember, so I'll do a quick smoke test around this before merging.

Reply:
Looks good as far as I can see, all the names are updating.